frontend/src/services/requestService.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+
+axios.defaults.headers.common = {
+    'X-Requested-With': 'XMLHttpRequest',
+    'Content-type': 'application/json',
+    'Accept': 'application/json'
+}
+
+const csrfToken = document.head.querySelector('meta[name="csrf-token"]');
```

Comment:
not sure if it's right way to get anything for axios params, such types of services shouldn't know anytthing about the DOM. Since if this service will be used in SSR it will fail